http/multipart/multipart.go

```diff
@@ -0,0 +1,57 @@
+package multipart
+
+import (
+	"errors"
+	"fmt"
+	ioutils "github.com/jfrog/gofrog/io"
+	"io"
+	"mime/multipart"
+	"os"
+)
+
+type FileWriterFunc func(fileName string) (io.WriteCloser, error)
+
+func ReadFilesFromStream(multipartReader *multipart.Reader, fileWriterFunc FileWriterFunc) error {
+	for {
+		// Read the next file streamed from client
+		fileReader, err := multipartReader.NextPart()
```

Comment:
Maybe we should close the fileReader as well

Reply:
I'm not sure @omerzi, see my comment https://github.com/jfrog/gofrog/pull/53/files#r1518793038